modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/LuaClientCodegen.java

```diff
@@ -138,8 +138,8 @@ public LuaClientCodegen() {
         importMapping.put("os", "io/ioutil");
 
         cliOptions.clear();
-        cliOptions.add(new CliOption(CodegenConstants.PACKAGE_NAME, "Lua package name (convention: lowercase).")
-                .defaultValue("swagger-client"));
+        cliOptions.add(new CliOption(CodegenConstants.PACKAGE_NAME, "Lua package name (convention: kebab-case).")
+                .defaultValue("openapi-client"));
```

Comment:
@daurnimator FYI. I've renamed the default value to "openapi_client" (similar to what we've done in other generators)

Reply:
Have you though? Seems to be using hypens rather than underscores?
Generally both hypens and underscores should *both* be avoided in lua module names. Anything after a hyphen should be a version number. Anything after a "_" is stripped if not found.
Thought you'll find lua modules in the wild that break this: we shouldn't encourage it.

Reply:
@daurnimator I picked kebab-case (e.g. openapi-client) to start with as I checked https://luarocks.org/ and found packages (top download) named with hyphens.

So what's the suggested naming convention for Lua package name? Just `openapiclient`?

Reply:
@daurnimator thanks for the clarification. I've updated the wording via https://github.com/OpenAPITools/openapi-generator/pull/1236/commits/11280274d99e48851fa7b4d35489ae5f326a96de